rotkehlchen/tests/api/test_premium.py

```diff
@@ -116,6 +117,7 @@ def test_get_premium_devices(rotkehlchen_api_server: APIServer) -> None:
                 'device_identifier': '43434343',
             },
         ],
+        'current_device_id': machineid.hashed_id(rotki.premium.username),
```

Comment:
So a question here. Isn't this one ofthe two above? I suppose it should be one of the list of devices? If so then can you specify it by making the current_device_id a variable and use it in both places?